docs/products/opensearch/howto/connect-with-python.rst

```diff
@@ -0,0 +1,44 @@
+Connect to OpenSearch cluster with Python
+=========================================
+
+You can interact with your cluster with the help of the `Python OpenSearch client <https://github.com/opensearch-project/opensearch-py>`_. 
+It provides a convenient syntax to send commands to your OpenSearch cluster. Follow its ``README`` file for installation instructions.
+
+To connect with your cluster, you need the **Service URI** of your OpenSearch cluster. You can find the connection details in the `**Overview** <https://console.aiven.io/project/dev-sandbox/services>`_ page. Alternatively, you can find it in Aiven command line interface `service command <https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get>`_. ``service_uri`` contains credentials, therefore should be treated with care.
```

Comment:
There is a layout issue with **. also, the link is not leading to the overview page, but to the list of services...

Reply:
thanks, replaced the link. can you clarify what is the layout issue?